package.json

```diff
@@ -2,11 +2,11 @@
   "name": "@vectara/react-search",
   "version": "0.0.9",
   "description": "A Vectara-powered Search component",
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.esm.js",
-  "types": "dist/index.d.ts",
+  "main": "lib/index.cjs.js",
+  "module": "lib/index.esm.js",
+  "types": "lib/index.d.ts",
```

Comment:
Changed this only because importing from 

```@vectara/react-search/lib/ReactSearchNext```

...IMO looks better than

```@vectara/react-search/dist/ReactSearchNext```